tracer/test/Datadog.Trace.Tests/Util/HexStringTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿// <copyright file="HexStringTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Util;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Util;
+
+public class HexStringTests
+{
+    [Theory]
+    [InlineData("0000000000000000", 0)]
+    [InlineData("00000000075bcd15", 123456789)]
+    [InlineData("ffffffffffffffff", 18446744073709551615)]
```

Comment:
These might be easier to confirm as correct if you used hex literals 😄 
```suggestion
    [InlineData("0000000000000000", 0)]
    [InlineData("00000000075bcd15", 0x75bcd15)]
    [InlineData("ffffffffffffffff", 0xffffffffffffffff)]
```